daemon/cluster/executor/container/adapter.go

```diff
@@ -65,17 +66,44 @@ func (c *containerAdapter) pullImage(ctx context.Context) error {
 	}()
 
 	dec := json.NewDecoder(pr)
+	dec.UseNumber()
 	m := map[string]interface{}{}
+	spamLimiter := rate.NewLimiter(rate.Every(1000*time.Millisecond), 1)
```

Comment:
@dperny I'm fine either way I think, but saw the extra dependencies for implementing the rate-limit, so thought that print every xx percent would possibly be a simpler solution.